library/src/hooks/useTimetable.tsx

```diff
@@ -23,19 +22,25 @@ function useTimeTable<T extends BaseTask>({
 }: UseTimeTableOption<T>) {
   const timeTableRef = useRef<HTMLDivElement | null>(null);
 
-  const taskListWithAutoVerticalPosition = setTaskListWithAutoVerticalPosition(
-    taskList,
+  // sorting
+  const sortedTaskList = taskList.sort(
+    (a, b) => a.startTime.getTime() - b.startTime.getTime(),
+  );
```

Comment:
가장 최말단 훅이니 좀더 예쁘게 함수로 감싸서 작성해준다면 sort하는 부분도 함수로 빼주는 것은 어떤가요?

```suggestion
  // sorting
  const sortedByStartToEndTaskList = (inputTaskList) => {
    return inputTaskList.sort(
      (a, b) => a.startTime.getTime() - b.startTime.getTime(),
    );
  };

  const sortedTaskList = sortedByStartToEndTaskList(taskList);
```
위 sorted가 필요한 이유가 그룹핑을 할 때 보다 group을 순차적으로 늘려가감에서 필요한 것이라면 해당 함수가 `groupByOverlappingTimesRange`안에 들어가는 것은 어떨까 싶기도 합니다!

Reply:
변경했습니다!